docs/quest_template.md

```diff
@@ -251,347 +377,245 @@ Like all flag based fields, **QuestFlags** can be added for the different types
+### RewardFactionValue5
 
-### OfferRewardEmoteDelay
 
-Emote delay in ms
+### RewardFactionOverride5
 
-### WDBVerified
 
-## **Examples dealing with quests**
+### TimeAllowed
 
-Always use PrevQuestId before using NextQuestId. NextQuestId is considered optional and to be used only when PrevQuestId is not sufficient
 
-### Basic quest
+### AllowableRaces
 
-Single, stand-alone quest with no prerequisites
 
-```
-    questA
-```
+### LogTitle
 
-```sql
-entry = questA        PrevQuestId = 0        NextQuestId = 0        ExclusiveGroup = 0        NextQuestInChain = 0
-```
+Title of the quest.
 
-### Prequest
+### LogDescription
 
-When this quest require another quest to be rewarded
+Objectives of the quest. If empty, quest is an auto-complete quest that can be immediately finished without accepting it first.
 
-```
-    questA
-```
+### QuestDescription
 
-```sql
-entry = questA        PrevQuestId = questX   NextQuestId = 0        ExclusiveGroup = 0        NextQuestInChain = 0
-```
+The quest text. You can use certain placeholders that will be filled in in-game: $B - line break, $N - name, $R - race, $C - class, $Gmale:female; (male and female can be replace with any synonymn you want, but the order must stay the same. IE: boy:girl / man:woman / sir:madam / dude:chick)
 
-### Chain of quests
+### AreaDescription
 
-Player get quests in a strict chain that must be completed in a specific order.
 
-```
-    questA
-      |
-    questB
-      |
-    questC
-      |
-    questD
-```
+### QuestCompletionLog
 
-```sql
-entry = questA      PrevQuestId = 0          NextQuestId = 0       ExclusiveGroup = 0       NextQuestInChain = questB
-entry = questB      PrevQuestId = questA     NextQuestId = 0       ExclusiveGroup = 0       NextQuestInChain = questC
-entry = questC      PrevQuestId = questB     NextQuestId = 0       ExclusiveGroup = 0       NextQuestInChain = questD
-entry = questD      PrevQuestId = questC     NextQuestId = 0       ExclusiveGroup = 0       NextQuestInChain = 0
-```
+Text sent to player when the player tries to talk to the NPC with the quest active but incomplete. (The text under the "Progress" title in Wowhead.) You can use certain placeholders that will be filled in in-game: $B - line break, $N - name, $R - race, $C - class, $Gmale:female; (male and female can be replace with any synonymn you want, but the order must stay the same. IE: boy:girl / man:woman / sir:madam / dude:chick)
+
+### RequiredNpcOrGo1
+### RequiredNpcOrGo2
+### RequiredNpcOrGo3
+### RequiredNpcOrGo4
 
-### Chain of quests with multiple start quests.
+Value &gt; 0:required creature\_template ID the player needs to kill/cast on in order to complete the quest.
+Value &lt; 0:required gameobject\_template ID the player needs to cast on in order to complete the quest.
+If\*RequiredSpellCast\*is != 0, the objective is to cast on target, else kill.
+NOTE: If RequiredSpellCast is != 0 and the spell has effects Send Event or Quest Complete, this field may be left empty.
```

Comment:
what is &gt &lt? I have never seen that before?

Reply:
I don't really know, that's how it was in the original file. Much of the information was copied from the previous file. Remember that I don't know English, my only goal was to update the file.